src/Umbraco.Web/umbraco.presentation/umbraco/preview/PreviewContent.cs

```diff
@@ -208,7 +204,11 @@ private static void DeletePreviewFile(int userId, FileInfo file)
             }
             catch (Exception ex)

```

Comment:
We should be catching a specific exception type here, not just Exception since we know which type we're expecting